x-pack/filebeat/module/suricata/eve/_meta/fields.yml

```diff
@@ -1,5 +1,6 @@
 - name: eve
   type: group
+  default_field: false
```

Comment:
To avoid changing the behavior for the existing fields can you mark this on the two new field groups instead.